data/index.html

```diff
@@ -15,35 +15,35 @@
             }
         </script>
         <style type="text/css">
-            html {  margin-right: 8px; 	margin-left: 8px; 	font-family: Helvetica, Arial, sans-serif;}
-            .links-container { 	padding-bottom: 4px; 	padding-top: 4px; 	display: block; 	text-align: right;}
-            .link1 { padding-bottom: 4px; padding-top: 4px; text-decoration: none; color: #006e00;}
+            html { margin-right: 8px; margin-left: 8px; font-family: Helvetica, Arial, sans-serif;}
+            .links-container { padding-bottom: 4px; padding-top: 4px; display: block; text-align: right;}
+            .link1 { padding:8px; text-decoration: none; color: #006e00; background-color: rgba(211, 211, 211, 0.2); border-radius: 10px; transition: background-color 0.5s;}
```

Comment:
There is a lot of redundant coding here; can the common attributes be moved to a style that is inherited, so that `.link1` and `.link2` differ only in `color`?